test/integration/rules/aria-required-children/aria-required-children.html

```diff
@@ -76,8 +76,18 @@
   <div aria-busy="true"></div>
 </div>
 <div role="suggestion" id="fail10" aria-busy="false"></div>
+<div role="list" id="fail11">
+  <li>Item 1</li>
+  <span role="link">Item 2</span>
+</div>
+<div role="list" id="fail12">
+  <div aria-live="polite">
+    <div role="listitem">List item 1</div>
+    <div role="listitem">List item 2</div>
+  </div>
+</div>
```

Comment:
I would like to see some more tests thrown at this. Looking at the code, i don't think these work yet.

```suggestion
</div>

<div role="list" id="pass14">
  <script></script>
  <style></style>
  <canvas hidden></canvas>
  <div><div role="listitem">Item 1</div></div>
  <div role="generic"><div role="listitem">item 2</div></div>
  <div role="presentation"><div role="listitem">item 2</div></div>
  <div role="none"><div role="listitem">item 2</div></div>
</div>

<div id="radiogroup" id="pass15">
  <div role="heading" aria-level="2">Heading</div>
  <ul>
    <li><div role="radio"></div></li>
    <li><div role="radio"></div></li>
  </ul>
</div>
```

Reply:
We want to handle this by removing the `requiredOwned` from the `radiogroup` role and allowing `menu` as a required child of `menubar`.